src/alpaca.erl

```diff
@@ -411,4 +411,24 @@ throw_with_variables_test() ->
     ?assertThrow({not_equal, 1, 2}, M:assert_equal(1, 2)),
     code:delete(M).
 
+record_update_test() ->
+    Files = ["test_files/update_record.alp",
+             "test_files/use_update_record.alp"],
+    [M1, M2] = compile_and_load(Files, []),
+
+    ?assertEqual(#{'__struct__' => record, x => 5, b => 2}, M2:main(unit)),
```

Comment:
I find it a bit confusing to use `unit` as the parameter in a call to an alpaca function `let main () = ...`, when the actual representation of unit is the empty tuple `()` (or `{}` in Erlang syntax). Speaking of this, shouldn't "let main () = " be compiled to a pattern match against `{}`, so this would crash?